content/learn/fundamental-wellbeing.md

```diff
@@ -1,73 +1,96 @@
 ---
-title: "Fundamental Wellbeing"
-created: 2023-04-21
-authors: 
+title: An Introduction to Dr. Jeffery Martin's Work on Fundamental Wellbeing
+created: 2023-11-01
+authors:
   - rufuspollock
   - jefferymartin
+description: Fundamental Wellbeing is the the experience of persistent, higher states of conscious, also known as awakening, enlightenment and other terms. This page is an introduction to the concept.
+image: /assets/images/fundamental-wellbeing-dr-jeffery-martin.jpg
 ---
-## What is Fundamental Wellbeing
+ At Life Itself, we believe that cultivating being is one of the core tenants of creating a wiser, weller world. Greater spiritual enlightenment, leads to better wellbeing, greater compassion, and honed abilities to be connected to ourselves and others.
+ 
+ While there are many terms for spiritual enlightenment, here we refer to the term Fundamental Wellbeing. In fact, this page is dedicated to Dr. Jeffery Martin's work, the man who coined the term and has spent over a decade researching how people reach Fundamental Wellbeing. 
+ 
+ By the end of this page you should have a solid understanding of Dr. Martin's work, the concepts of Fundamental Wellbeing and how to explore it further.
 
-Fundamental Wellbeing is a termed coined (or made popular) by [Jeffery Martin](https://lifeitself.org/notes/jeffery-martin). In more academic settings he tends to refer to it as Persistent Non-Symbolic Experience (Martin credits the "non-symbolic" terminology to [abc/@cook-greuter](https://lifeitself.org/abc/@cook-greuter)).
+## Interviews with Dr. Jeffery Martin
 
-The terminology is a secular (and hence friendlier) and standardized label for [state](https://lifeitself.org/state) stages. Martin's map of the transcendent section of state space is the most detailed and worked out of any we know of. It greatly extends [Wilber](https://lifeitself.org/Wilber)'s map and those of classic wisdom traditions.
+Before we explore Fundamental Wellbeing throughout this page, begin by watching one of the most extensive interviews with Dr. Jeffery Martin on the web. 
```

Comment:
Before we do X ... is never great intro. Not needed and kind of blurb-y.

How about:

> The following interviews with Martin himself conducted in 2022 and 2023 provide an in-depth introduction to Martin's work o Fundamental Wellbeing.

Reply:
Before we do X provides a subconscious psychological cue in the reader that there is more to come after this section. However, not the most effective use of it here so switched to yours